scout/server/blueprints/variant/views.py

```diff
@@ -47,6 +47,7 @@ def update_tracks_settings():
     # update user in database with custom tracks info
     user_obj["igv_tracks"] = selected_tracks
     store.update_user(user_obj)
+    setattr(current_user, "igv_tracks", selected_tracks)
```

Comment:
Took me some time with this PR because if you are locally on demo and change the settings of the tracks that you want to see, for instance to show genes:

<img width="709" alt="image" src="https://github.com/user-attachments/assets/bbd0b5fe-b78c-4da7-83c0-d7723defdf1c">

It saves the correct track in the user document in the database, but jinja current users is not updated and still shows the previously selected track:

<img width="695" alt="image" src="https://github.com/user-attachments/assets/3cb1c051-bcf8-4602-be64-2315d2f8e44d">

Reply:
Oh, and we had some path to display that didn't involve reloading the user from db? I thought we did that every page reload? Well spotted!